src/pybind/mgr/dashboard/frontend/src/app/shared/components/critical-confirmation-modal/critical-confirmation-modal.component.html

```diff
@@ -13,7 +13,8 @@
         <ng-container *ngTemplateOutlet="bodyTemplate; context: bodyContext"></ng-container>
         <div class="question">
           <p i18n>Are you sure that you want to
-            {{ actionDescription | lowercase }} the selected {{ itemDescription }}?</p>
+            {{ actionDescription | lowercase }}
+            <span *ngIf="!itemName">the selected {{ itemDescription }}</span><strong *ngIf="itemName">{{ itemName }}</strong>?</p>
```

Comment:
Minor stuff: `ngIf/else`?

```suggestion
            <strong *ngIf="itemName; else noName">{{ itemName }}</strong><span #noName>the selected {{ itemDescription }}</span>?</p>
```

Reply:
Done